audioplayer.h

```diff
@@ -0,0 +1,41 @@
+#include <Windows.h>
+#include <MMSystem.h>
+#include <iostream>
+#include <io.h>
+#include <dirent.h>
+#include <vector>
+#include <string>
+#include <algorithm>
+
+
+#pragma comment(lib, "winMM.lib")
+
+using namespace std;
+
+
+class AudioPlayer {
+    public:
+        int trackNumber;
+        string trackTiming;
+        string trackName;
+        string fileLink;
+        string currentVolume = "50";
+        vector<string> files;
+        bool isPlaying = false;
```

Comment:
You should refactor this class a little bit. Right now all your methods receive no arguments and all fields are public. You change fields and then call methods. But it's easier and better (for incapsulation purposes) to pass those variables to methods.
https://stackoverflow.com/questions/14399929/should-i-use-public-or-private-variables

Bad:
```
cin >> musicPlayer.trackNumber;
musicPlayer.ChangeAudio()
```

Good:
```
int trackNumber;
cin >> trackNumber;
musicPlayer.ChangeAudio(trackNumber)
```

Every method of this call should be fixed.

Reply:
fixed, maybe not completely.